### PR TITLE
Update schema docs for GPU requests

### PR DESCRIPTION
**Pull request by eshiroma**
Only merge after #481 is released.

/gcbrun

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -844,6 +847,55 @@ for sufficient available resources, and new clusters will be created with
...
+Currently only Nvidia platforms are supported (requests entry key
+`nvidia.com/gpus`). See the
+[Compute Engine GPU guide](https://cloud.google.com/compute/docs/gpus)
+for a full list of available GPU platforms.
+
+Note that for `resource` entries with GPU requests:
+
+*   `resources` may contain at most one entry with GPU requests
+*   `affinity` and `replicas` are ignored
+    * `cpu` and `memory` may be defined in the same entry, but must be
```
Comment (huyhg) [inline docs/schema.md]: Rm. `cpu` and `memory` actually need replicas, otherwise we don't know how to fit.

Comment (eshiroma) [reply docs/schema.md]: Done.

File: docs/schema.md [hunk 2]
```diff
@@ -816,15 +816,21 @@ x-google-marketplace:
 
 Each entry under `resources` is roughly equivalent to a workload in the app.
 
-* `affinity` defines the relationship between the nodes and the replicas.
+*   `requests` defines the desired resource allocations, and is required for each
```
Comment (huyhg) [inline docs/schema.md]: Style is only a single space after `*`, right?

Comment (eshiroma) [reply docs/schema.md]: Updated - I'm not sure what the universal/general best practice is, but leaving as-is (single-space). Internally, autoformatters will correct to having text align at the fourth space.

**Pull request merged**